be/src/storage/rowset/segment_rewriter.cpp

```diff
@@ -2,8 +2,18 @@
 
 #include "segment_rewriter.h"
 
-namespace starrocks {
+#include "env/env.h"
+#include "gen_cpp/segment.pb.h"
+#include "gutil/strings/substitute.h"
+#include "storage/fs/block_manager.h"
+#include "storage/fs/fs_util.h"
+#include "storage/rowset/segment.h"
+#include "storage/rowset/segment_writer.h"
+#include "storage/vectorized/chunk_helper.h"
+#include "util/crc32c.h"
+#include "util/slice.h"
 
+namespace starrocks {
 namespace segment_v2 {
```

Comment:
segment_v2 namespace is already removed.

Reply:
done